examples/browserify/.gitignore

```diff
@@ -0,0 +1 @@
+node_modules
```

Comment:
I was more thinking that people will just copy those examples, but this won't work anyway, so feel free to leave it like that :-)